server/tests/backends/fixtures/filter/dates.json

```diff
@@ -0,0 +1,74 @@
+{
+  "exclude": [
+    "sql"
+  ],
```

Comment:
should we not exclude the sql translator?

Reply:
yeah, true, i forgot that !
Updated !

Reply:
My bad! I din't get that sql != snowflake
So it's OK to exclude it.
Sorry!